content/en/apps/reference/targets.md

```diff
@@ -20,30 +20,30 @@ All targets are defined in the `targets.js` file as an array of objects accordin
+| `goal` | `integer` | For targets with `type: 'percent'`, an integer from 0 to 100. For `type: 'count'`, any positive number. If there is no goal, put -1.                                                                                                                                                                                                                                                                                          | yes |
+| `appliesTo` | `'contacts'` or `'reports'` | Do you want to count reports or contacts? This attribute controls the behavior of other attributes herein.                                                                                                                                                                                                                                                                                                                    | yes |
+| `appliesToType` | If `appliesTo: 'reports'`, an array of form codes. If `appliesTo: 'contacts'`, an array of contact types. | Filters the contacts or reports for which `appliesIf` will be evaluated. For example, `['person']` or `['clinic', 'health_center']`. For example, `['pregnancy']` or `['P', 'pregnancy']`.                                                                                                                                                                                                                                    | no |
+| `appliesIf` | `function(contact, report)` | If `appliesTo: 'contacts'`, this function is invoked once per contact and `report` is undefined. If `appliesTo: 'reports'`, this function is invoked once per report. Return true to count this document. For `type: 'percent'`, this controls the denominator.                                                                                                                                                               | no |
+| `passesIf` | `function(contact, report)` | For `type: 'percent'`, return true to increment the numerator.                                                                                                                                                                                                                                                                                                                                                                | yes, if `type: 'percent'`. Forbidden when `groupBy` is defined. |
+| `date` | `'reported'` or `'now'` or `function(contact, report)` | When `'reported'`, the target will count documents with a `reported_date` within the current month. When `'now'`, target includes all documents. A function can be used to indicate when the document should be counted. Default is `'now'`.                                                                                                                                                                                  | no |
```

Comment:
You can also put something like this to add more clarity about what happens when it's a function:
 

> When this property is undefined or the value is null the default is `'now'`.

Reply:
sure !